### DISCUSSION TITLE
Github Student Developer Pack not approved by Github Education

### DISCUSSION THREAD

**Discussion by yash08123** [community/community#140361] [GitHub Education]
### Hi everyone,

**Hi everyone,**
I applied for github education and after a while it was accepted and approved. But, the education benefits aren't provided to me until now.
I've provided all my ID card but still it dosen't provide me the benifits

**Comment by davevad93** [comment]
Hi @yash08123 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @yash08123 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by yash08123** [reply]
![image](https://github.com/user-attachments/assets/54bae05b-6db4-42d7-a942-2562f9b1e3e3)

I've uploaded the Collage ID and Current Timetable both but still its rejected

**Reply by davevad93** [reply]
Sorry @yash08123 I misunderstood, from what you said earlier I thought you had been approved and were waiting for your benefits to become available,  make sure your academic documents show a full current date proving your current student status, also take a look at the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).
